<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WHoleMag</class>
 <widget class="QWidget" name="WHoleMag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="c_hole_type">
       <item>
        <property name="text">
         <string>Slot Type 50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 51</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 52</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 53</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 54</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 55</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 56</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="w_hole" native="true">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
